core/primitives-core/src/version.rs

```diff
@@ -145,6 +145,9 @@ pub enum ProtocolFeature {
     // Stateless validation: in statelessnet, shuffle shard assignments for chunk producers every
     // epoch.
     StatelessnetShuffleShardAssignmentsForChunkProducers,
+    /// Cross-shard congestion control according to NEP-539.
+    #[cfg(feature = "protocol_feature_nep539_congestion_control")]
+    Nep539CongestionControl,
```

Comment:
nit: How about just CongestionControl? 
I would get rid of the cargo feature as well, it's annoying and we can do it in runtime anyway.